src/components/Capture/index.js

```diff
@@ -148,7 +152,9 @@ class Capture extends Component {
   }
 
   onScreenshot = canvas => {
-    canvasToBase64Images(canvas, this.handleImage)
+    canvasToBase64Images(canvas, (lossyBase64, base64) => {
+      this.handleImage(lossyBase64, base64)
```

Comment:
if you convert the base64 image here, you don't need the complexity of `getFile`, since on this method it's always a base64 image and on `onImageFileSelected` it's always a file.